src/components/handbook-navigation/index.js

```diff
@@ -1,19 +1,42 @@
-import React from "react";
+import React, { useState } from "react";
 import { HiOutlineChevronLeft } from "react-icons/hi";
 import { Link } from "gatsby";
 import TOCWrapper from "./toc.style";
+import { IoMdClose, IoIosArrowDropdownCircle } from "react-icons/io";
 
 const TOC = () => {
+  const [expand, setExpand] = useState(false);
   return (
     <TOCWrapper>
       <div className="go-back">
         <Link to="/community/handbook">
           <HiOutlineChevronLeft />
           <h4>Table of Content</h4>
         </Link>
+        <div className="toc-toggle-btn">
+          {expand ? (
+            <IoMdClose
+              className="mobile-menu-icon open"
+              style={{ width: "1.5rem", height: "1.5rem", cursor:"pointer" }}
+              fill="#3c494f"
```

Comment:
This can be covered under the stylesheet as well. And, also use the generic code to fetch the color, i.e, from the props.
As done in this [line(https://github.com/layer5io/layer5/blob/258e5786e5726e4e01b6aece1fe04d3f7b18d385/src/sections/Company/Brand/brandPage.style.js#L10).

Reply:
Oh, i see. Thank you. I will fix it.